file-formats/intm/type/zif_aff_intm_v1.intf.abap

```diff
@@ -0,0 +1,111 @@
+INTERFACE zif_aff_intm_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Intelligent Scenario Model</p>
+    "! Intelligent scenario model
+    BEGIN OF ty_intelligent_model,
+      "! <p class="shorttext">Model Name</p>
+      "! Model name
+      name                 TYPE c LENGTH 30,
+      "! <p class="shorttext">Intelligent Scenario</p>
+      "! Intelligent scenario of the model
+      intelligent_scenario TYPE c LENGTH 20,
+      "! <p class="shorttext">Modelling Context ID</p>
+      "! Modelling context ID which is the parent of the model
+      modelling_context_id TYPE c LENGTH 36,
+      "! <p class="shorttext">Model Type</p>
+      "! Model type
+      model_type           TYPE c LENGTH 256,
+      "! <p class="shorttext">Adapter ID</p>
+      "! Adapter ID
+      adapter_id           TYPE c LENGTH 32,
+      "! <p class="shorttext">Native Model Type</p>
+      "! Native model type
+      native_model_type    TYPE c LENGTH 256,
+      "! <p class="shorttext">Model Specification</p>
+      "! Model specification
+      model_specification  TYPE string,
+      "! <p class="shorttext">Model Definition</p>
+      "! Model definition
+      model_definition     TYPE string,
+      "! <p class="shorttext">State</p>
+      "! State of the model
+      state                TYPE c LENGTH 10,
```

Comment:
Is there a limited set of possible values for state? Then you could make an enumeration for this field

Reply:
We will have other state, but this is under discussion. Also, we are hiding it in UI. If needed we will do it in next scope.